lib/lorekeeper/json_logger.rb

```diff
@@ -95,6 +95,13 @@ def inspect
 
     private
 
+    # Some instrumentation libraries pollute the stacktrace and create a large output which may
+    # cause problems with certain logging backends.
+    # Hardcoring newrelic now here. In the future if this list grows, we may make it configurable.
+    def clean_backtrace(backtrace)
+      backtrace.reject{ |line| line.include?("/newrelic_rpm-") }
```

Comment:
[nits] missing space after `reject`

btw would it make sense to make that newrelic string a constant?

Reply:
updated

Reply:
mmm is the interepreter smart enough to see the array is a size 1 constant and inline the whole thing? with your new fix I'm afraid it's going to hit performance a bit to do that extra `any?` call for every line when we know right now there's only one item. How about keeping it as is for now and use `any?` if/when there's an extra pattern to add?

Reply:
reverted to not use any?